### ISSUE TITLE
toMatchSnapshot fails to work

### ISSUE THREAD

**Issue by ronlawrence3** [briebug/jest-schematic#14]
**Describe the bug**
Adding a toMatchSnapshot() assertion in a test on a fresh angular 6 project gets an error.

**To Reproduce**
With @angular/cli@6:
```
ng new testproj
cd testproj
ng add @briebug/jest-schematic
# edit app.component.spec.ts and add a toMatchSnapshot() assertion
npm test
```
Error message is:
```
$ npm test

> fooa@0.0.0 test /tmp/fooa
> jest

 FAIL  src/app/app.component.spec.ts
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    src/app/app.component.spec.ts:22:23 - error TS2339: Property 'toMatchSnapshot' does not exist on type 'Matchers<any>'.

    22     expect(app.title).toMatchSnapshot();
                             ~~~~~~~~~~~~~~~

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        4.592s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

**Expected behavior**
I expected these to work once project is converted to jest

**Desktop (please complete the following information):**
 - OS: Linux
 - Node version: 8.11.3
 - NPM or Yarn version NPM 
 - App type Angular CLI
 - Angular version ^6.1.0

**Comment by ronlawrence3** [comment]
I think the key to fixing this is to replace `jasmine` with `jest` in the tsconfig.spec.json file, and add @types/jest. (and possibly remove @types/jasmine from the package.json) I will see if I have time to submit a patch if this works..

**Comment by rgunczer** [comment]
@ronlawrence3 replacing jasmine with jest seems to work. However in VSCode TS still complains. Eg.
expect(service.getPersons).toBeInstanceOf(Function);
Property 'toBeInstanceOf' does not exist on type 'Matchers<any>'.

**Comment by ronlawrence3** [comment]
It looks like with a fresh project from the latest cli, and the latest schematic from this project, things seem to work for me now.

But as @rgunczer  says, VScode still doesn't like it. if I add jest to the main tsconfig.json, it is ok with it, so vscode must be using that tsconfig.

**Comment by t1m4lc** [comment]
A quick fix :
```
const fixture = TestBed.createComponent(TodoComponent);
(<any>expect(fixture)).toMatchSnapshot();
```
you can create an helper ` const snap = (fixture: any): any => (<any>expect(fixture)).toMatchSnapshot();
`
And use it : `snap(fixture);`

**Comment by schuchard** [comment]
Please see if `ng add @briebug/jest-schematic@3.0.0-0` solves the issue

**Comment by schuchard** [comment]
This should be fixed with the latest 3.0.0 release. Please reopen if not.
